src/components/Admin/ListDevices/index.js

```diff
@@ -41,30 +42,32 @@ class ListDevices extends React.Component {
             device.lastLoginIP !== undefined ? device.lastLoginIP : '-';
           const lastActive =
             device.lastActive !== undefined
-              ? new Date(device.lastActive).toDateString()
+              ? moment(new Date(device.lastActive)).format(
+                  'MMMM Do YYYY, H:mm:ss',
+                )
               : '-';
           macIdArray.forEach(macId => {
             const device = devices[macId];
             let deviceName = device.name !== undefined ? device.name : '-';
-            deviceName =
-              deviceName.length > 20
-                ? deviceName.substr(0, 20) + '...'
-                : deviceName;
             let location = 'Location not given';
             if (device.geolocation) {
               location = (
                 <span
                   onClick={this.handleClick}
                   name={macId}
-                  style={{ cursor: 'pointer' }}
+                  style={{ cursor: 'pointer', color: '#4285f4' }}
                 >
-                  {device.geolocation.latitude},{device.geolocation.longitude}
+                  {device.geolocation.latitude}
```

Comment:
Can we add a `,` here?

Reply:
Also, the hyperlink color of View | Edit | Delete is different from location color.